packages/opds-browser/src/components/OPDSBrowser.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { createStore, applyMiddleware } from "redux";
+let thunk: any = require("redux-thunk");
+import reducers from "../reducers/index";
+import Root from "./Root";
+
+export default class OPDSBrowser extends React.Component<RootProps, any> {
+  store: Redux.Store;
+  root: any;
+  static App: any;
```

Comment:
if we typed this correctly it would be a circular import. that might be an argument for putting the component under the app instead.